docs/pages/comment_parsing.rst

```diff
@@ -5,12 +5,17 @@ Comment Parsing
 
 A common task for many bots and scripts is to parse a submission´s comments. In
 this tutorial we will go over how to do that as well as talking about comments
-in general. To illustrate the problems, we'll write a small script that replies
+in general.
+
+The Basics
+----------
+
+To illustrate the problems, we'll write a small script that replies
```

Comment:
With "the problems" separated from the block above it's not clear what "the problems" are. Can you you provide some clarity?